src/translations/en.json

```
@@ -116,6 +116,7 @@
     "Language": {
       "Title": "Language",
       "English": "English",
+      "Furry": "Furry",
```

Languages titles ought to be in the language they represent, to help people choose the right one. The current title may cause confusion.

```suggestion
      "Furry": "Fuwwy",
```